config/config.yaml

```diff
@@ -9,9 +9,9 @@ policy:
     # features act as flags whose presence/absence can be used to render
     # a common set of features across branch(es) For eg: there are some
     # branches that support el/7, and some that doesn't - to render el/7
-    # support on these branches, add `el7` as a feature of these
+    # support on these branches, add `el7-pgo-build` as a feature of these
```

Comment:
This comment does not explain the meaning of this feature correctly any more.

Reply:
pls review